pom.xml

```diff
@@ -78,11 +78,6 @@
         <artifactId>json</artifactId>
         <version>20230227</version>
       </dependency>
-      <dependency>
-        <groupId>org.springframework</groupId>
-        <artifactId>spring-jcl</artifactId>
-        <version>6.0.9</version>
-      </dependency>
```

Comment:
Recent versions of the Jenkins core BOM (not to be confused with the Jenkins plugin BOM), which is imported in the Jenkins plugin parent POM, import the Spring BOM. So there is no need to manage Spring versions anywhere except in Jenkins core.